### PR TITLE
[Init] Clean up implementation

### PR DESCRIPTION
**Pull request by avanderhoorn**

### CODE REVIEW

**Review by mike-kaufman — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -69,7 +69,15 @@ function preInit(initPromise) {
     dataView.preInit(initPromise);
 }
 
-function init(promise, resolve, reject) {
+function init() {
+    // setup init promise
+    let resolve;
+    let reject;
```
Comment (davidkpiano) [inline src/index.js]: You can stick the `onTimeout` (and `setTimeout`) fn/calls inside the promise to avoid having to declare these variables.

Comment (avanderhoorn) [reply src/index.js]: Not following... code snippet?

Comment (davidkpiano) [reply src/index.js]: https://gist.github.com/davidkpiano/51d2d9c5f55a93c7cdb70f7a42532c09

**Review by davidkpiano — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (davidkpiano) [reply src/index.js]: None of the `postRender` functions actually use the `promise` so this can be refactored/simplified.

**Review by davidkpiano — APPROVED** [review]

**Pull request merged**